pages/_app.js

```diff
@@ -39,7 +39,7 @@ export default function App({ Component, pageProps }) {
     event.preventDefault();
     const formData = new FormData(event.target);
     const data = Object.fromEntries(formData);
-    if (data.review.trim().length === 0) return;
+    if (!data.review.trim()) return;
```

Comment:
The `{ .. }` around `if` have value as well, but I forgive you this time!

Reply:
can u pls explain why is it better to have the `{..}`